nautobot/circuits/views.py

```diff
@@ -102,6 +103,31 @@ class ProviderUIViewSet(NautobotUIViewSet):
     queryset = Provider.objects.all()
     serializer_class = serializers.ProviderSerializer
     table_class = tables.ProviderTable
+    object_detail_content = ObjectDetailContent(
+        panels=(
+            ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields="__all__",
+                exclude_fields=["name", "comments"],
```

Comment:
TODO: 
 Missing Fields: `Circuits` - circuits_count
 Missing Feature: `Customer Portal` is not hyperlinked